db/migrate/20180919152423_create_quote_types.rb

```diff
@@ -0,0 +1,11 @@
+class CreateQuoteTypes < ActiveRecord::Migration[5.2]
+  def change
+    create_table :quote_types do |t|
+      t.string :name, limit: 255, null: false
+      t.string :order, null: false
```

Comment:
Isn't it the QuoteStatus? QuoteType should be capable to organize the order of their quote status and the 'change events' related to them

Reply:
I put in both. Can have to.